### PR TITLE
6416 - LEGAL API - Court Order and Plan of Arrangement

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#6416

*Description of changes:*

- Added columns to the model;
- Added migration;
- Added test;
- Added columns to lear_db.sql.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: legal-api/tests/unit/models/test_filing.py [hunk 1]
```diff
@@ -25,20 +25,27 @@
...
     ANNUAL_REPORT,
     CHANGE_OF_DIRECTORS,
     CORRECTION_AR,
+    COURT_ORDER,
     FILING_HEADER,
     SPECIAL_RESOLUTION,
 )
 from sqlalchemy_continuum import versioning_manager
 
-from legal_api.exceptions import BusinessException
```
Comment (thorwolpert) [inline legal-api/tests/unit/models/test_filing.py:37]: why were these moved?
the linter should complain about the move

File: legal-api/tests/unit/models/test_filing.py [hunk 2]
```diff
@@ -672,3 +679,20 @@ def test_linked_not_correction(session):
     # test
     assert filing1.json['filing']['header']['isCorrected'] is False
     assert filing2.json['filing']['header']['affectedFilings'] is not None
+
+
+def test_alteration_filing_with_court_order(session):
```
Comment (thorwolpert) [inline legal-api/tests/unit/models/test_filing.py:684]: Can you add a test to ensure if any validators are necessary they work?

**Review by thorwolpert — COMMENTED** [review]
is there no validation required other than schema correctness?

File: legal-api/tests/unit/models/test_filing.py (hunk 2 above)
Comment (andrepestana-aot) [reply legal-api/tests/unit/models/test_filing.py:684]: I don't know why but my Flake8 local is slightly different from the GH's action. I moved because Flake8 complianed. How can I make it use the same rules?

File: legal-api/tests/unit/models/test_filing.py (hunk 1 above)
Comment (andrepestana-aot) [reply legal-api/tests/unit/models/test_filing.py:37]: I don't know why but my Flake8 local is slightly different from the GH's action. I moved because Flake8 complianed. How can I make it use the same rules?

Comment (thorwolpert) [reply legal-api/tests/unit/models/test_filing.py:37]: are you on Windows+WSL or Mac/Linux? running ```make setup``` should get it all install correctly
if you are not, then make sure you do a ```pip install -e .``` in the services home directory, so it knows these modules are local and not external

Comment (andrepestana-aot) [reply legal-api/tests/unit/models/test_filing.py:37]: I'm using WSL 2. I used to run ```make setup``` but I also used to run ``` pip install .```. I see now that I cannot run pip after running make. I fixed the import. The strange is that the checks on github passed. Thanks!

**Comment by andrepestana-aot** [comment]
> is there no validation required other than schema correctness?
I added another test to check saving invalid data. Is that what you mean?

**Comment by andrepestana-aot** [comment]
I don't understand why codecov is failing. It was passing before the latest PR was merged.

File: legal-api/tests/unit/models/test_filing.py (hunk 2 above)
Comment (andrepestana-aot) [reply legal-api/tests/unit/models/test_filing.py:686]: I added another test that checks what happened when trying to insert invalid data to the columns.

**Review by thorwolpert — APPROVED** [review]
